### PR TITLE
Red black tree issue #91

### PR DESCRIPTION
**Pull request by ManavLohia945**
## 🛠️ Issue (Number)

Issue no #91

## 👨‍💻 Changes proposed
None yet 

## ✔️ Check List (Check all the applicable boxes) 

- [ ✔️] My code follows the code style of this project.
- [ ✔️] This PR does not contain plagiarized content.
- [✔️ ] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
NA

## 📷 Screenshots
image

### CODE REVIEW

**Comment by ManavLohia945** [comment]
Ok will do that by tomm

**Comment by Kumar-laxmi** [comment]
@ManavLohia945 For this PR don't update the **README.md**

**Comment by ManavLohia945** [comment]
i have now not updated the README file as instructed by you

**Comment by Kumar-laxmi** [comment]
Something is wrong with the **README.md** in this PR, please look into it

**Comment by Kumar-laxmi** [comment]
> Something is wrong with the **README.md** in this PR, please look into it

@ManavLohia945 Please remove the updation of **README.md** from this PR

**Review by zugzwang03 — CHANGES_REQUESTED** [review]
Add your approach in a  few comment lines

**Comment by ManavLohia945** [comment]
@Kumar-laxmi I am so confused as to what should I include in the README.md file. If you tell me exactly what to include in that file it would be appreciated

**Comment by ManavLohia945** [comment]
@Kumar-laxmi sir please check this PR. I am very sorry for the delay. I hope the content in the README.md file is correct now .

**Review by Sanjanabharadwaj25 — CHANGES_REQUESTED** [review]
image
@ManavLohia945 The README is not correct. Make the necessary changes.

**Comment by ManavLohia945** [comment]
Done plz check @Sanjanabharadwaj25

File: README.md [hunk 1]
```diff
@@ -36,6 +36,58 @@
...
+| 51.  | Binary Tree to BST | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Trees/BinaryTreeToBST.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/BinaryTreeToBST.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Trees/BinaryTreeToBST.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/BinaryTreeToBST.py) | Trees |
+| 52.  | Morris Traversal | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Trees/morris_transversal.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/MorrisTraversal.cpp) | Trees |
+| 53.  | Red-Black Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/red_black.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/red_black.py) | Trees |
 | 2.  | Banker's Algorithm | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Deadlock%20Avoidance/bankers.c), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Deadlock%20Avoidance/bankers_algorithm.py) | Deadlock Avoidance |
```
Comment (Sanjanabharadwaj25) [inline README.md:91]: @ManavLohia945 Please resolve this.

**Comment by Sanjanabharadwaj25** [comment]
> Done plz check @Sanjanabharadwaj25

image
There are still some corrections to be made. Please look into it.

**Comment by ManavLohia945** [comment]
Could you send me the link i have to add in the readme file in the red black tree section @Sanjanabharadwaj25

**Comment by ManavLohia945** [comment]
Now check @Sanjanabharadwaj25

**Review by zugzwang03 — APPROVED** [review]

**Comment by Kumar-laxmi** [comment]
@ManavLohia945 You can better close this PR and open a fresh PR with no changes done in the README.md

**Comment by ManavLohia945** [comment]
Okay I will keep the readme file as it is in the main branch

**Pull request closed without merging**